examples/tensorgraph/test_mnist.py

```diff
@@ -1,115 +0,0 @@
-import unittest
```

Comment:
It might be good to keep these tests. They test more than the ipython notebook does.